### ISSUE TITLE
Enhancement: Collect LevelUp Awards

### ISSUE THREAD

**Issue by OverlordQ** [NecronomiconCoding/NecroBot#530]
When you level up, the bot doesn't collect the level up awards so the first time you connect with a device if you've leveled any significant amount, you'll blow way past your inventory cap.

There's probably an RPC call somewhere to trigger this.

**Comment by Aristrine** [comment]
I agree with this, it past by the Cap by at least 1/2. If it doesn't ensue a ban however, I have no qualms.

**Comment by tw88** [comment]
if they update the api this is avaiable, together with fancy stuff like gym battling :yum:

**Comment by DeR0X** [comment]
there will be no gym battling

**Comment by tw88** [comment]
why? source? its in the API

**Comment by DeR0X** [comment]
sure it is in the API but he said he wont destroy the game. So we wont have this feature in this Bot

**Comment by cawk** [comment]
Will work on this soon (the get rewards not the gym function)

**Comment by cawk** [comment]
https://gyazo.com/0f8265721664ae0adc7321b312e2355b
it works properly am just doing tidy ups so it doesnt look like this on output
